extension/chrome/elements/compose-modules/compose-draft-module.ts

```diff
@@ -111,23 +121,43 @@ export class ComposeDraftModule extends ViewModule<ComposeView> {
           sendable.body['text/plain'] = `[cryptup:link:draft_compose:${this.view.draftId}]\n\n${sendable.body['text/plain'] || ''}`;
         }
         const mimeMsg = await sendable.toMime();
-        if (!this.view.draftId) {
-          const { id } = await this.view.emailProvider.draftCreate(mimeMsg, this.view.threadId);
+        if (!this.view.draftId || this.isLocalDraftId(this.view.draftId)) {
```

Comment:
If I understand this code correctly, if I'm loading a draft from storage and I'm now back online, it will save the local draft to the cloud. Is that correct? That would be a good move, and may be good to add a short comment. Maybe this part could actually be moved into a separate method, this `draftSave` is growing big. Something like `private doUploadDraftWithLocalStorageFallback`

Reply:
> If I understand this code correctly, if I'm loading a draft from storage and I'm now back online, it will save the local draft to the cloud. Is that correct?

Correct.

> That would be a good move, and may be good to add a short comment.

Done :+1:

> Maybe this part could actually be moved into a separate method, this draftSave is growing big. Something like `private doUploadDraftWithLocalStorageFallback`

Done 👍